fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -2304,6 +2295,9 @@ struct ShardedRocksDBKeyValueStore : IKeyValueStore {
 						writeBatch.DeleteRange(ps->cf, toSlice(cRange.begin), toSlice(cRange.end));
 					}
 				}
+			} else if (format == RocksDBKeyValues) {
+				a.done.sendError(not_implemented());
```

Comment:
Is this actually pending implementation? Or this will not be supported?

For RocksDBKeyValues, will the restore be handled here or this is supposed to be "not supported"?

Reply:
It will be implemented in a following PR(#9030)